README.md

```diff
@@ -21,7 +21,29 @@ in case you would like to use the `NN_Trainer`.
 
 ## How to use
 
-You may find examples of gapfilling a genome scale reconstruction (GEM) with `dnngior` with a complete or a defined medium in this [example notebook](tutorials/example.ipynb).
+Gapfilling models is done using the `Gapfill` class:
+```python
+import dnngior.gapfill_class.Gapfill  
+Gapfill(path_to_model)
+```
+
+You may find examples of gap-filling a genome scale reconstruction (GEM) with `dnngior` with a complete or a defined medium in this [example notebook](tutorials/example.ipynb). `dnngior` can gapfill both ModelSEED and BiGG models, to gapfill BiGG models you need to specify modeltype. 
+
+```python
+Gapfill(path_to_BiGG_model, modeltype='BiGG')
+```
+
+## Custom Networks
+
+By default `dnngior` uses an universally trained network capable of accurate predictions under most circumstances. If desired, it is possible to change the Neural Network you want to use during gapfilling:
+
+```python
+Gapfill(path_to_model, trainedNNPath=path_to_NN)
+```
+
+You can train your own Neural Network following this tutorial: [example training NN](tutorials/NN_training_example.ipynb).
+
+Alternatively you can find additional custom Neural Networks for several taxonomic groups: [Custom Networks](docs/NN/custom_networks/). Upon request additional specially trained networks can be made available for specific biomes or taxonomic groups.
 
 
 ## License
```

Comment:
consider adding a point to the license file already on the repo

Reply:
since dnngior  is already on biorxiv 
consider pointing at it on the cite section	https://www.biorxiv.org/content/10.1101/2023.07.10.548314v1